### ISSUE TITLE
Feature Request: Zoom Function

### ISSUE THREAD

**Issue by stareagle** [stuffmatic/fSpy#87]
Would it be possible to add a zoom function rather than just pressing the shift key when aligning the x, y, z lines. Perhaps using the scroll wheel.

**Comment by kalamazandy** [comment]
Yes, this would help a LOT. During the current preview, the lines are hidden also, so it's difficult to know the effects of the small changes. If we could have simple zoom (and pan) then it would be much easier to be precise where needed.

**Comment by ygorcsouza** [comment]
Yes, please!
